libyul/backends/wasm/WasmDialect.cpp

```diff
@@ -27,13 +27,32 @@
 using namespace std;
 using namespace solidity::yul;
 
+namespace {
+
+// These are not YulStrings because that would be too complicated with regards
+// to the YulStringRepository reset.
+
+std::string const i64{"i64"};
```

Comment:
If you name this `c_i64` or `s_i64` (as per coding style) then you don't have to change `i64` below.

Reply:
Otherwise, `i64_type` should actually be `i64Type`